meta-mel/conf/local.conf.append.ls1021atwr

```diff
@@ -0,0 +1,2 @@
+UBOOT_CONFIG_ls1021atwr = "sdcard"
+EXTRA_IMAGEDEPENDS += "virtual/bootloader"
```

Comment:
This should be `EXTRA_IMAGEDEPENDS_append_ls1021atwr = " virtual/bootloader"`, and we should likely add comments above these lines indicating their purpose to clarify for the user, since this will end up visible to them in their local.conf. Again see the minnow one for an example.